.github/workflows/comment-on-e2e-success.yml

```diff
@@ -17,11 +23,31 @@ jobs:
             Check the results here: https://github.com/gnosis/safe-react-e2e-tests/actions/runs/${{ github.event.client_payload.run_number }}
             ${{ github.event.client_payload.message }}
       - name: Create failure comment
-        if: ${{ github.event.client_payload.status == 'failure' }}
+        if: ${{ github.event.client_payload.status == 'failure' && steps.checkComment.outputs.comment-id == '' }}
         uses: peter-evans/create-or-update-comment@v1
         with:
           issue-number: ${{ github.event.client_payload.pr_number }}
           body: |
             E2E Tests Failed
             Check the results here: https://github.com/gnosis/safe-react-e2e-tests/actions/runs/${{ github.event.client_payload.run_number }}
             ${{ github.event.client_payload.message }}
+      - name: Edit success comment
+        if: ${{ github.event.client_payload.status == 'success' && steps.checkComment.outputs.comment-id != ''}}
+        uses: peter-evans/create-or-update-comment@v1
+        with:
+          issue-number: ${{ github.event.client_payload.pr_number }}
+          comment-id: ${{ steps.checkComment.outputs.comment-id }}
```

Comment:
Maybe if you just pass this id to the "Create success comment" job, you won't need to duplicate the jobs?
If the comment id is empty, I think it would just create a new comment?

Reply:
Yes, in the docs says that you could split the jobs of create or update if necessary, but I would test to leave the `comment-id` undefined and see how it works to avoid duplication

https://github.com/peter-evans/create-or-update-comment#where-to-find-the-id-of-a-comment

Also, by default according to the docs it will append content to the original comment as `edit-mode` parameter by default is set to `append`. Maybe in this case we prefer to use the `replace`

https://github.com/peter-evans/create-or-update-comment#action-inputs